components/authz-service/server/v2/policy.go

```diff
@@ -902,3 +922,8 @@ func (s *policyServer) setVersionForInterceptorSwitch(v api.Version) {
 		s.vChan <- v
 	}
 }
+
+func (s *policyServer) isBeta2p1() bool {
```

Comment:
This falls apart when we're using IAM v2 endpoints with authz-service running on v1. This is what our inspec tests have been doing for a while now.